business-registry-dashboard/app/stores/affiliations.ts

```diff
@@ -42,10 +42,15 @@ export const useAffiliationsStore = defineStore('brd-affiliations-store', () =>
   }
 
   async function removeBusiness (payload: RemoveBusinessPayload) {
-    // If the business is a new registration then remove the business filing from legal-db
-    if (payload.business.corpType.code === CorpTypes.INCORPORATION_APPLICATION) {
+    // If the business is a new IA, amalgamation, continuation in, or registration then remove the business filing from legal-db
+    if ([
+      CorpTypes.INCORPORATION_APPLICATION,
+      CorpTypes.AMALGAMATION_APPLICATION,
+      CorpTypes.CONTINUATION_IN,
+      CorpTypes.REGISTRATION
+    ].includes(payload.business.corpType.code)) {
       const filingResponse = await getFilings(payload.business.businessIdentifier)
-      if (filingResponse && filingResponse.status === 200) { // TODO: fix typing
```

Comment:
This `filingResponse.status` check is not needed as it's not returned. So, the previous code never actually entered this block.